src/deque.jl

```diff
@@ -74,18 +73,40 @@ type Deque{T}
     Deque() = Deque{T}(DEFAULT_DEQUEUE_BLOCKSIZE)
 end
 
+"""
+    deque(::Type{T})
+
+Create a deque of type `T`
```

Comment:
Should also mention that `blksize` can be passed to set blocksize (see constructor above). Anyway, this needs to be symmetric with the `Queue` docstring.